packages/env/lib/env.js

```diff
@@ -69,12 +70,15 @@ module.exports = {
 			// Preemptively start the database while we wait for sources to download.
 			dockerCompose.upOne( 'mysql', {
 				config: config.dockerComposeConfigPath,
+				log: config.debug,
```

Comment:
Was thinking it would be useful if we logged the exact `docker-compose` command that `wp-env` is running. Is there a way we can make `dockerCompose` do that?

Reply:
I don't think so, we would have to extend it.